cou/commands.py

```diff
@@ -538,6 +541,8 @@ def parse_args(args: Any) -> CLIargs:  # pylint: disable=inconsistent-return-sta
     if len(args) == 0 or (len(args) == 1 and args[0] == "help"):
         parser.print_help()
         parser.exit()
+        # This line is reachable in unit tests when .exit() is mocked.
+        return  # type: ignore[unreachable]
```

Comment:
I don't understand why are you adding a return here. I think this always will be unreachable with `parser.exit()`

Reply:
I added a comment above that line to explain:

> This line is reachable in unit tests when .exit() is mocked.

It was hard figuring out how best to test the function. I tried a few methods including various refactorings of the function. But the logic is all intertwined.

Reply:
If you have any suggestions for refactoring or improving testability, please let me know :)

Reply:
Adding `return` seems overkill. I also cannot find any error related to `.exit` being mocked

Reply:
I don't like it either, but if you remove the `return`, the unittests will fail. I'm happy for suggestions on how to improve the tests so we don't need this. Maybe a mock can raise systemexit, so it won't change the control flow but we can stil verify it was called? That function seems difficult to test. :/

Reply:
> Maybe a mock can raise systemexit

Aha, turns out this was key. I've reworked the tests, and the function no longer needs the return statements. :)